app/models/cloud_database.rb

```diff
@@ -0,0 +1,10 @@
+class CloudDatabase < ApplicationRecord
+  include NewWithTypeStiMixin
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :cloud_tenant
+  belongs_to :flavor
+
+  serialize :extra_attributes
```

Comment:
We need to standardize on a name for this to be consistent across the models.  The Middleware team used "properties" which I kind of like, but I also like the "extra_" bit here.

Thoughts?  @matthewd @chessbyte @blomquisg